main.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
```

Comment:
I haven't seen yet what else we can add here.
But the question is: 
> Is there anything else other than `required_providers` that we could add here?

I guess we will get to know in the future anyways :sweat_smile:

Reply:
I haven't seen anything else. But yes, I guess we'll see